pkg/reconciliation/reconcile_datacenter.go

```diff
@@ -70,6 +70,8 @@ func (rc *ReconciliationContext) ProcessDeletion() result.ReconcileResult {
 				// Exiting to let other parts of the process take care of the decommission
 				return result.Continue()
 			}
+			// How could we have pods if we've decommissioned everything?
+			return result.RequeueSoon(5)
```

Comment:
Are you adding the requeue here to handle the scenario where `len(dcs) == 1`?

Reply:
Why would we requeue in that case? If len(dcs) == 1, we go and delete the DC.

Reply:
It's not obvious to me why the need for the requeue here. Can you explain?